Remotion/Web/Development.WebTesting/WebTestFeatureCollectionWithFallback.cs

```diff
@@ -0,0 +1,132 @@
+﻿// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+
+namespace Remotion.Web.Development.WebTesting;
+
+/// <summary>
+/// Version of <see cref="WebTestFeatureCollection"/> with support for a <see cref="IReadOnlyWebTestFeatureCollection"/> as fallback.
+/// The fallback is not modified by this collection.
+/// </summary>
+/// <remarks>
+/// The fallback is assumed to be immutable and thread-safe.
+/// If the fallback requires mutation only add operations are considered safe.
```

Comment:
```suggestion
/// If the fallback requires mutation, only add operations are considered safe.
```
"requires mutation"? the interface is readonly. The current type can't know whats in there, not sure what sentence implies.